circuit_maintenance_parser/parsers/globalcloudxchange.py

```diff
@@ -0,0 +1,85 @@
+"""Circuit Maintenance Parser for Equinix Email Notifications."""
+
+import re
+from datetime import datetime
+from typing import Any, Dict, List
+
+from bs4.element import ResultSet  # type: ignore
+
+from circuit_maintenance_parser.parser import EmailSubjectParser, Html, Status
+
+
+class HtmlParserGcx1(Html):
+    """Custom Parser for HTML portion of Global Cloud Xchange circuit maintenance notifications."""
+
+    def parse_html(self, soup: ResultSet) -> List[Dict]:
+        """Parse an Global Cloud Xchange circuit maintenance email.
+
+        Args:
+            soup (ResultSet): beautiful soup object containing the html portion of an email.
+
+        Returns:
+            Dict: The data dict containing circuit maintenance data.
+        """
+        data: Dict[str, Any] = {"circuits": []}
+
+        for div in soup.find_all("div"):
+            for pstring in div.strings:
+                search = re.search("Dear (.*),", pstring)
+                if search:
+                    data["account"] = search.group(1)
+
+        # Find Circuits
+        for table in soup.find_all("table"):
+            for row in table.find_all("tr"):
+                cols = row.find_all("td")
+                if len(cols) == 2 and "Service ID" not in cols[0].text:
+                    impact = "NO-IMPACT"
```

Comment:
you could use the impact enum, for this and the next one.
BTW, the default impact should be OUTAGE?